src/common/UserAvatar.js

```diff
@@ -48,16 +53,20 @@ function UserAvatar(props: Props) {
     <View>
       <Touchable onPress={onPress}>
         <View>
-          <Image
-            source={{
-              uri: avatarUrl.get(PixelRatio.getPixelSizeForLayoutSize(size)).toString(),
-              ...(avatarUrl instanceof FallbackAvatarURL
-                ? { headers: getAuthHeaders(auth) }
-                : undefined),
-            }}
-            style={style}
-            resizeMode="cover"
-          />
+          {!isMuted ? (
+            <Image
+              source={{
+                uri: avatarUrl.get(PixelRatio.getPixelSizeForLayoutSize(size)).toString(),
+                ...(avatarUrl instanceof FallbackAvatarURL
+                  ? { headers: getAuthHeaders(auth) }
+                  : undefined),
+              }}
+              style={style}
+              resizeMode="cover"
+            />
+          ) : (
+            <IconUserMuted size={size} color={color} />
```

Comment:
I notice that the Font Awesome icon isn't square; it looks like its height is getting set to 48px while its width is greater, which disturbs the alignment with the 48x48 avatars:

![image](https://user-images.githubusercontent.com/22248748/118032801-0fd2a100-b336-11eb-8bfd-25500cf95433.png)

I guess that's a design choice that Font Awesome has made, hmm. My sense from [a bit of reading](https://material.io/design/iconography/system-icons.html#design-principles) is that some icon libraries use a square for the boundaries of each icon? Alya pointed out that we'll probably want to use a user icon without the slash. That may let us choose an icon from a different library, where they're all nice and square.

Being short and wide is more inconvenient than being tall and narrow. The underlying component that gets used is a `Text`, as we've discussed elsewhere in this PR. I don't yet know how to scale down the contents of a `Text` by specifying a max width, letting the height take a computed value, without cutting stuff off. The `size` prop (which is used as the `Text`'s `style.fontSize`) seems to control the height. If the icon were tall and narrow (from some experimentation, I think Font Awesome's ['user' icon](https://fontawesome.com/icons/user) is this way), I think we could set the height to 48 using `size` and just make sure it's centered in a 48x48 square (I have some working code for this centering if you'd like me to share).

Reply:
Good catch! `user` has the same problem but with the reverse aspect ratio, I've fixed it by centering the text, let me know if you'd prefer a different approach.